hathor/manager.py

```diff
@@ -995,6 +1008,9 @@ def step_validations(self, txs: Iterable[BaseTransaction]) -> None:
         """ Step all validations until none can be stepped anymore.
         """
         assert self.tx_storage.indexes is not None
+        if not self.tx_storage.indexes.deps:
+            return
```

Comment:
Is it safe to just skip it? Or should we exit the process? Anyway, we should at least log that we are skipping this step.

Reply:
I just did that, it now only calls `_resume_validation` and `step_validations` if sync-v2 is enabled.

Reply:
This was moved to #275